telethon_generator/parser/tl_object.py

```diff
@@ -173,6 +173,9 @@ def __init__(self, name, arg_type, generic_definition):
     def __str__(self):
         # Find the real type representation by updating it as required
         real_type = self.type
+        if self.flag_indicator:
+            real_type = '#'
```

Comment:
In theory, flag arguments don't need to be called exactly `flags` (imagine you had 33 flagged arguments, you would need two integer flags to mark them).